biomage/experiment/info.py

```diff
@@ -0,0 +1,247 @@
+            FROM user_access WHERE experiment_id = '{experiment_id}'
+    """
+
+    try:
+        users = db(query)
+        return _get_user_cognito_info(users, env)
+    except:
+        return []
+
+
+def _get_experiment_samples(db, experiment_id):
+    query = f"""
+        SELECT id as sample_id, name, sample_technology, options \
+            FROM sample WHERE experiment_id = '{experiment_id}'
+    """
+
+    try:
+        return db(query)
+    except:
+        return []
+
+
+def _get_experiment_runs(db, experiment_id):
+    query = f"""
+        SELECT pipeline_type, execution_arn, last_status_response \
+            FROM experiment_execution WHERE experiment_id = '{experiment_id}'
+    """
+
+    try:
+        return db(query)
+    except:
+        return []
+
+
+def _print_tabbed(key, value):
+    print(f"{key}\t\t: {value}")
+
+
+def _format_item(details):
+    for key, value in details.items():
+        _print_tabbed(key, value)
+
+
+def _format_table(content):
+
+    header = list(content[0].keys())
+    table = []
+
+    for item in content:
+        table.append(list(item.values()))
+
+    print(tabulate(table, header, tablefmt="simple"))
+
+
+def _format_runs(content):
+    for run in content:
+        print(run['pipeline_type'].upper())
+        _print_tabbed('execution_arn', run['execution_arn'])
+
+        run_details = run['last_status_response'][run['pipeline_type']]
+
+        _print_tabbed('status\t', run_details['status'])
+        if(run_details['status'] != "SUCCEEDED"):
+
+            error_string = f"{run_details['error']['error']}"
+            if(run_details['error'].get('cause')):
+                error_string += run_details['error']['cause']
+
+            _print_tabbed('error\t', error_string)
+
+        _print_tabbed('start_date', run_details['startDate'])
+        _print_tabbed('stop_date', run_details['stopDate'])
+        _print_tabbed('completed_steps', run_details['completedSteps'])
+        print()
+
+
+def _pretty_print(result):
+
+    WIDTH = 30
+
+    print()
+    print("=" * WIDTH , "EXPERIMENT", "=" * WIDTH, "\n")
+    _format_item(result['info'])
+    print()
+
+    print("=" * WIDTH , "USERS", "=" * WIDTH, "\n")
+    _format_table(result['users'])
+    print()
+
+    print("=" * WIDTH , "SAMPLES", "=" * WIDTH, "\n")
+    if len(result['samples']):
+        _format_table(result['samples'])
+    else:
+        print("No samples uploaded")
+    print()
+
+    print("=" * WIDTH , "PIPELINE RUNNER", "=" * WIDTH, "\n")
+    _format_item(result['pipeline_runner'])
+    print()
+
+    print("=" * WIDTH , "RUNS", "=" * WIDTH, "\n")
+    if len(result['runs']):
+        _format_runs(result['runs'])
+    else:
+        print("Experiment has not been processed")
+
+
+@click.command()
+@click.option(
+    "-e",
+    "--experiment_id",
+    required=True,
+    help="Experiment ID to be copied.",
+)
+@click.option(
+    "-i",
+    "--input_env",
+    required=True,
+    default=STAGING,
```

Comment:
considering it's just an information command, make the default `production` as it will be our most usual use case

Reply:
martin also commented on this, and he thinks because the other commands uses `STAGING`, we'd expect this one to use staging by default too. I agree with him. Otherwise, we can make this not have default so users will have to input the environment explicitly.